src/pybind/mgr/volumes/fs/nfs.py

```diff
@@ -13,12 +13,12 @@
 class GaneshaConf(object):
     # pylint: disable=R0902
 
-    def __init__(self, cluster_id, rados_pool, rados_namespace, mgr):
-        self.mgr = mgr
-        self.cephx_key = ""
-        self.cluster_id = cluster_id
-        self.rados_pool = rados_pool
-        self.rados_namespace = rados_namespace
+    def __init__(self, nfs_conf):
```

Comment:
@mgfritch would `class GaneshaConf` be helpful for you?

Reply:
Yes! Let's look at re-using this where possible in #33581